### ISSUE TITLE
Private messages to my XMPP address duplicate every 5 minutes with the same timestamp

### ISSUE THREAD

**Issue by Mikaela** [siacs/Conversations#2214]
#### General information

* **Version:** eg 1.12.2

1.15.2, but has been happening with older versions for months

* **Device:** eg Google Nexus 5

OnePlus 3

* **Android Version:** eg Android 6.0 Stock or Android 5.1 Cyanogenmod

Android version would be 7.0 and I am not sure on stock as the phone did not come with OxygenOS community build, it still comes from the manufacturer. OxygenOS version would be OP3_O2_Open_10, but has been also happening on older versions.

* **Server name:** eg conversations.im, jabber.at or self hosted

joindiaspora.com

* **Server software:** ejabberd 16.04 or prosody 0.10 (if known)

Prosody 0.9.9

* **Installed server modules:** eg Stream Managment, CSI, MAM

https://github.com/iNPUTmice/ComplianceTester/blob/master/reports/joindiaspora.com.txt

* **Conversations source:** eg PlayStore, PlayStore Beta Channel, F-Droid, self build (latest HEAD)

PlayStore Beta Channel

#### Steps to reproduce

1. Receive a message from anyone in private discussion (initiated with your XMPP address).
2. Wait for five minutes
3. The message duplicates with the same timestamp
4. Repeat to get the message for the third time.

#### Expected result

I expect Conversations to show and notify me of the message only once, not hear beep every five minutes until I disable notifications for eight hours and later see the message seven times.

#### Debug output

Please post the output of adb logcat. The log should begin with the start of Conversations and include all the
steps it takes to reproduce the problem.

````
adb logcat -s conversations
````

**WILL BE DONE SOON**

**Comment by Mikaela** [comment]
This issue has refused to happen within logcat for 40 minutes, I hope this can still be kept open even if with some label about missing logcat as in the Conversations MUC another user said to have the same issue.

I hope they will come here and have better luck with logcat.

**Comment by Mikaela** [comment]
Correction, server is https://github.com/iNPUTmice/ComplianceTester/blob/master/reports/joindiaspora.com.txt

Compliance Tester run wanted to remove jappix.com.txt and I probably got distracted by it and linked to wrong file. 

I would edit this issue, but GitHub mobile UI is missing the button.

**Comment by Mikaela** [comment]
More information: I found option "Archiving preferences" and noticed it was set to Never and am now trying with "Always" as MAM was thought to be the culprit earlier for all messages appearing with the same timestamp. 

* * * * *

Below lines from MUC earlier today:

* * * * *

I have been using Conversations more again and I still experience duplicated messages. 
The guess on it having something to do with MAM is probably accurate as when I click notifications, I get "no more history on the server". 
I am less annoyed now that I have found Android setting "notify silently" and enabled it with Conversations, but I will have to uncheck it later if someone replaces Tox with XMPP

>  ciblia: specificity is the soul of native. Your statement doesn't help anybody if you don't say if that's for muc messages or regular messages and if the message is actually duplicate in the conversation view or if you just get notified a bunch of times. Also it would be helpful to know what server and what sending client was involved

iNPUTmice: regular, message is duplicated with the same timestamp. Prosody IIRC and client has been BitlBee and Conversations. https://github.com/siacs/Conversations/issues/2214 has more details including ComplianceTester links to joindiaspora.com and the issue has also happened when messaging myself or betseen joindiaspora.com and Kapsi.fi,I don't know which I did.

Ciblia@joindiaspora.com messaged mikaela@kapsi.fi and receives duplicates of own messages while kapsi.fi sees the message only once.

> ciblia: so the sending client is bitlbee?

> Even with mam it should repeat itself at max once
 If it is not send multiple times

> If it is happening without Conversations starting a new session (you can see that in the account details) it is the sending client just sending the message multiple times

iNPUTmice: repeating clients are any that message me, even receiving Conversations itself

* * * * *

After toggling the setting, I messaged contact whom with this has happened before and cross-messaged both of my XMPP accounts, so I think if this fixed the issue, I won't see new duplicates after returning from shower and there won't be a problem when I message from other clients. 

If the issue was resolved, I think Conversations is doing something wrong by using MAM when it's configured (by default not to) or it doesn't keep log on when it accessed MAM and downloads the same messages over and over.
